qa/tasks/vstart_runner.py

```diff
@@ -762,6 +762,23 @@ def find_remote(self, daemon_type, daemon_id):
         """
         return LocalRemote()
 
+    def get_keyring(self, client_id):
```

Comment:
Can this be inherited via ceph_manager.py?

Reply:
Yes, this was supposed to updated after my "inherit" got merged.

Done, now. Thanks.